packages/core/src/plugins/web3FormAssets/web-3-form.css

```diff
@@ -0,0 +1,12 @@
+input,
+select,
+textarea {
```

Comment:
One last nit, do the Bootstrap form classes work well here or would it be more cumbersome than using CSS? https://getbootstrap.com/docs/5.1/forms/overview/#overview

Reply:
Mmm they definitely work but if bootstrap is used the problem is for custom inputs (ie not using `name-input` or `message-input` etc) they would not be styled the same way since users may not have added the bootstrap class. 

Eg. something like this 
```
<web-3-form>
  <label for="fname">First name:</label><br>
  <input type="text" id="fname" name="fname"><br>
  <label for="lname">Last name:</label><br>
  <input type="text" id="lname" name="lname">
</web-3-form>
```

I can change it to add the bootstrap class for the custom inputs if that is better?

Reply:
Sorry I missed this! I see, in that case the Bootstrap classes are quite cumbersome - it seems that CSS is a better solution 👍